.packit.yaml

```
@@ -0,0 +1,48 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: foreman-installer.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - foreman-installer.spec
+    - .packit.yaml
```

Please keep yamllint happy :)
```suggestion
  - foreman-installer.spec
  - .packit.yaml
```